tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Where did this file come from? Or how was it generated?

Reply:
Came from [here](https://github.com/OneSignal/react-onesignal/blob/34ef57e4002795dd4cbfbf869c3880fec15ad28e/tsconfig.json) (the original version before the 2 release).